plugin/goto.py

```diff
@@ -34,15 +34,20 @@ def run(
         point: Optional[int] = None,
         side_by_side: bool = False,
         force_group: bool = True,
-        fallback: bool = False
+        fallback: bool = False,
+        group: int = -1
```

Comment:
Should also add this argument to `is_enabled` above (even if it works without it).